controls/forms.py

```diff
@@ -88,3 +88,32 @@ class ImportOSCALComponentForm(forms.Form):
         required=False
     )
     json_content = forms.CharField(label='OSCAL (JSON)', widget=forms.Textarea())
+
+class ImportProjectForm(forms.Form):
+
+    file = forms.FileField(label="Select project file (.json)",
+        widget=forms.FileInput(
+            attrs={
+                'onchange': "fillJSONContent(this);",
+                'accept':'application/json'
+            }
+        ),
+        required=False
+    )
+    json_content = forms.CharField(label='Project (JSON)', widget=forms.Textarea())
```

Comment:
Might as well benefit off the other forms that do the same thing! I like it. It makes me wonder if we should have a base "ImportForm" class, and then extend from that to make our separate "Component" and "Project" importers. But that's out of scope of this task. It's probably a TODO down the line. Just a thought.